tests/rptest/services/admin_ops_fuzzer.py

```diff
@@ -605,4 +605,7 @@ def check():
                 )
             return False
 
-        wait_until(check, timeout_sec=timeout, backoff_sec=2)
+        try:
+            wait_until(check, timeout_sec=timeout, backoff_sec=2)
+        finally:
+            self.stop()
```

Comment:
please make stop idempotent e.g. by introducing a flag and setting it at the end of the method and checking in the beginning. at the moment it at least print history twice